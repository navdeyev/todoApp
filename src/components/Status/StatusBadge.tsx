import {Paragraph} from 'components/Styled/Styled';
import {defaultTheme} from 'components/Styled/themes';
import {rem} from 'components/Styled/utils';

const StatusBadge = Paragraph.extend`
    flex: 1 1 0px;
    display: flex;
    cursor: pointer;
    align-items: center;
    padding: 0 ${ rem(5) };
    border-radius: ${ rem(3) };
    border: 1px solid ${ props => props.theme.statusBorderColor };
    justify-content: center;
    
    &:hover {
      border: 1px solid ${ props => props.theme.hoverStatusBorderColor };
    }
`;

StatusBadge.defaultProps = { theme: defaultTheme};
StatusBadge.displayName = 'StatusBadge';

export default StatusBadge;