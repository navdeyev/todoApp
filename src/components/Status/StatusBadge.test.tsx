import {shallow} from 'enzyme';
import * as React from 'react';

import StatusBadge from './StatusBadge';

describe('StatusBadge', () => {
    it('renders a component', () => {
        const props = {};
        const render = shallow(<StatusBadge {...props}/>);
        expect(render.getElement()).toMatchSnapshot();
    });
});