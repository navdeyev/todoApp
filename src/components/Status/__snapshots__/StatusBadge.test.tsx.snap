// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBadge renders a component 1`] = `
<p
  className="sc-bxivhb gayghH"
/>
`;
